### ISSUE TITLE
Update to laminas/ServiceManager?

### ISSUE THREAD

**Issue by vwroad87** [akrabat/rka-slim-zfsm-container#10]
It would appear that Zend Service Manager has changed hands or something. It is now called Laminas as notified by PHP composer:
> Package zendframework/zend-servicemanager is abandoned, you should avoid using it. Use laminas/laminas-servicemanager instead.

I hope this project is not abandoned since it seems to work very well.

Best regards.

**Comment by akrabat** [comment]
Yeah - Zend Framework has been renamed Laminas as the project moved to the Linux Foundation.

We should definitely update this.
